epicfocus/README.md

```diff
@@ -0,0 +1,129 @@
+# Epic Focus: Measure your way to a better time to market
+
+> "The successful warrior is the average man, with laser-like focus."
+> - Bruce Lee
+
+There are several recurring wishes our clients bring to us, one of which is speed, shorter time to market. However, there is no dial that we can turn to deliver value faster. Software teams are not like cars; there's no acceleration pedal. Even if we try to speed up by adding more resources, in many cases, the actual bottleneck will just become more apparent.
+
+In our search for increased delivery of value, we hunt for these bottlenecks. No two contexts are the same, and for this story, we have a particular context in mind. Symptoms in different organizations are often similar, and our story might apply to your setting if you recognize the problems we encountered.
+
+## Observing the symptoms
+Like doctors, we always start by observing the symptoms of the bottlenecks that can slow down time to market. There might be numerous signs one can notice in an organization. We would like to bring your attention to three we encountered:
+1. Perception of slowness
+2. Lack of collaboration
+3. Team has nothing to present
+
+### Perception of slowness
+Speed is a new currency. If you are fast, chances are you are going to be the first on the market with that new exciting feature. It is not about being better; it is about getting there first. Everyone is consumed by what's new. Few people are interested in what's better.
+
+Therefore, the lack of value delivered is noticed quite quickly. While competitors are already delivering features that are on your future roadmap, you still struggle to answer what value was added to your product last quarter or even last year. Although you might have done a lot of work, if it did not result in tangible and significant customer-facing changes, the perception might be that nothing was changed at all.
+
+### Lack of collaboration
+Collaboration is the action of working with someone to produce something. In working together, there is an exchange of ideas; new and enhanced ideas are born. It allows you to deliver the best designs and the best software. Collaboration is a source of creativity and innovation; it promotes healthy employee relationships. It also creates a faster feedback loop and allows issues to be found and resolved earlier in the process.
+
+At the heart of collaboration is communication. Look at how people around you communicate, do they prefer face-to-face communication, or are email and slack their goto options? Do you see people working on one computer? Do they look like a team?
```

Comment:
Slack

Reply:
> people working on one computer
do you mean pair programming here?

Reply:
> communicate, do they prefer
I would put a full stop here instead of a comma.

Reply:
I mean pair programming, but when you are observing things, this is what you see - people sitting behind one computer. Is it unclear?

Reply:
An engineer can guess that it's pair programming, but for a non-engineer it might be unclear why two people are behind one computer. But I guess it's ok, you can leave it as is.